libclc/cmake/modules/AddLibclc.cmake

```diff
@@ -122,7 +122,7 @@ macro(add_libclc_builtin_set arch_suffix)
           --long-width=${long_width}
           --char-signedness=${signedness}
           "$<TARGET_PROPERTY:prepare-${obj_suffix},TARGET_FILE>"
-          DEPENDS "prepare-${obj_suffix}" libclc-remangler )
+          DEPENDS "${builtins_obj_path}" libclc-remangler )
```

Comment:
Please, merge with the sycl branch. This is already committed there.

Reply:
Oh I forgot to drop this commit before pushing, I just rebased, it should be good now.